libs/libc/string/lib_memcpy.c

```diff
@@ -34,7 +36,7 @@
  * Name: memcpy
  ****************************************************************************/
 
-#ifndef CONFIG_LIBC_ARCH_MEMCPY
+#if !defined(CONFIG_LIBC_ARCH_MEMCPY) || (LIBC_BUILD_FUNCTION)
```

Comment:
#if defined(LIBC_BUILD_FUNCTION) && !defined(CONFIG_LIBC_ARCH_MEMCPY)
and other similar place